src/main.rs

```diff
@@ -97,25 +97,22 @@ The normal workflow is:
             .takes_value(false))
         .subcommand(SubCommand::with_name("create")
             .about("Creates a yarner project in the current directory")
-            .arg(Arg::with_name("file")
-                .help("The base file for the doc sources, with normal file extension, but without additional style extension.")
-                .value_name("file")
+            .arg(Arg::with_name("main_file")
+                .help("The main file for the document sources, with normal file extension, but without additional style extension.")
```

Comment:
"style" can now be replaced by "md" or "Markdown".

Reply:
Yeah, this is what was triggering my comment on suffix handling for Markdown files. Will fix once we decide on that...

Reply:
Replaced "style" by "Markdown" for now because I fear the other discussion is sprawling beyond what this PR should try to achieve...